build.gradle

```diff
@@ -14,14 +14,15 @@ buildscript {
     ext.annotationVersion = "1.3.0"
     ext.androidxCoreVersion = "1.8.0"
     ext.mockwebserverVersion = "4.2.0"
+    ext.gradleMavenPublish = '0.22.0'
```

Comment:
FFTI but I wonder if we should keep this version where it was since it's only used in that one place... Just think that ideally we only have versions that are in multiple modules here.

Reply:
sounds good!